pkg/issuer/acme/dns/dns_test.go

```diff
@@ -58,12 +122,12 @@ func TestSolverFor(t *testing.T) {
 					KubeObjects: []runtime.Object{
 						newSecret("cloudflare-key", map[string][]byte{
 							"api-key": []byte("a-cloudflare-api-key"),
-						}),
+						}, fakeIssuerNamespace),
```

Comment:
These tests were passing before and after the change. I only parameterised the Secret namespace here to clarify how the namespaces are correlated.